tests/test_scheduler.py

```diff
@@ -18,6 +17,50 @@ def test_scheduler():
     assert actual == expected
 
 
+def test_scheduler_lsf():
+
+    expected = """#BSUB -J abcd
+#BSUB -n 48
+#BSUB extra_stuff 12345"""
+
+    props = {
+        "scheduler": "lsf",
+        "job_name": "abcd",
+        "extra_stuff": "12345",
+        "nodes": "12",
+        "tasks_per_node": "4",
+    }
+
+    js = JobScheduler.get_scheduler(props)
+    actual = js.job_card.content()
+
+    assert actual == expected
+
+
+def test_scheduler_pbs():
+
+    expected = """#PBS cpus 2
+#PBS -l select=4:mpiprocs=4:ompthreads=4:ncpus=16:mem=8gb"""
+
+    props = {
+        "scheduler": "pbs",
+        "nodes": "4",
+        "threads": "12345",
+        "cpus": "2",
```

Comment:
What are `cpus`? And how is this used above?

Reply:
leftover from trying to fill in the gaps mapping these.